app/controllers/instruments_controller.rb

```diff
@@ -1,6 +1,6 @@
 class InstrumentsController < ProductsCommonController
 
-  customer_tab :show, :public_schedule
+  customer_tab :show, :public_schedule, :list
```

Comment:
This is definitely a place where I'm not crazy about the mixing of admin and regular user paths in the same controller. It would be really nice to split that out at some point. What if we called this action `public_list` just to clarify its role? It can still route to `/list`.